extensions/jdbc/jdbc-mysql/jdbc-mysql-deployment/src/main/java/io/quarkus/jdbc/mysql/deployment/JDBCMySQLDBProcessor.java

```diff
@@ -0,0 +1,23 @@
+package io.quarkus.jdbc.mysql.deployment;
+
+import io.quarkus.deployment.annotations.BuildStep;
+import io.quarkus.deployment.builditem.FeatureBuildItem;
+import io.quarkus.deployment.builditem.NativeEnableAllCharsetsBuildItem;
+import io.quarkus.deployment.builditem.substrate.SubstrateResourceBuildItem;
+
+public class JDBCMySQLDBProcessor {
+    @BuildStep
+    FeatureBuildItem feature() {
+        return new FeatureBuildItem(FeatureBuildItem.JDBC_MYSQL);
+    }
+
+    @BuildStep
+    SubstrateResourceBuildItem resource() {
+        return new SubstrateResourceBuildItem("com/mysql/cj/util/TimeZoneMapping.properties");
```

Comment:
So you got it to work in the end?

Reply:
Yes. The native tests and native executable works fine locally. I’ll look to add the integration test on CI.